crates/trippy-tui/src/frontend/render/bar.rs

```diff
@@ -15,18 +15,18 @@ pub fn render(f: &mut Frame<'_>, rect: Rect, app: &TuiApp) {
         Protocol::Icmp => format!(
             "{}/{}",
             fmt_target_family(app.tracer_config().data.target_addr()),
-            t!("ICMP"),
+            "ICMP",
         ),
         Protocol::Udp => format!(
             "{}/{}/{}",
             fmt_target_family(app.tracer_config().data.target_addr()),
-            t!("UDP"),
+            "UDP",
             app.tracer_config().data.multipath_strategy(),
         ),
         Protocol::Tcp => format!(
             "{}/{}",
             fmt_target_family(app.tracer_config().data.target_addr()),
-            t!("TCP"),
+            "TCP",
         ),
```

Comment:
Arguably these abbreviations are fixated and accepted in other languages, no need to translate?

Reply:
I think this is fine, I put the following in the 0.12.0 release note:
> Many common abbreviated technical terms, such as IPv4 and ASN, are not translated.

I'd say if we are going to not attempt to translate these technical abbreviations then we should do so for all of them (so, everything in the current locale file where all languages use the same english abbreviation.  I can compile a list.  It feels like an orthogonal topic so I'd be tempted to do this as a separate PR, either before or after this one.